jaeger/util.go

```diff
@@ -1,26 +1,29 @@
 package jaeger
 
 import (
-	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"net/url"
 	"strconv"
 
 	"github.com/kiali/kiali/config"
-	"github.com/kiali/kiali/log"
 	"github.com/kiali/kiali/models"
 )
 
-func prepareQuery(u *url.URL, namespace, app string, query models.TracingQuery) {
-	queryApp := app
-	if config.Get().ExternalServices.Tracing.NamespaceSelector && namespace != config.Get().IstioNamespace {
-		queryApp = fmt.Sprintf("%s.%s", app, namespace)
+type JaegerServiceName = string
```

Comment:
In other code areas we have discuss to avoid to map primitive types.
It's fine to deal directly with string.
Otherwise code gets populated with a lot of abstractions.
Also, in the code q.Set("service", jsn) is expecting a string, too, so abstraction is weak here.

Reply:
I'm puzzled because I thought the compiler would expect explicit type conversion when using string or something derived from string, but it seems it doesn't. Not sure why, sometimes it does, sometimes not. My intention is to avoid future possible errors where we would call `prepareQuery` with something not formatted as a jaeger service name. But if the compiler doesn't enforce explicit type conversion, it is meaningless... so I'm fine to remove.